scraper/scraper.py

```diff
@@ -15,30 +15,27 @@
   use_ssl=True, 
   ca_certs=certifi.where())
 
-# ignore 400 cause by IndexAlreadyExistsException when creating an index
-es.indices.create(index='test-index', ignore=400)
```

Comment:
https://kb.objectrocket.com/elasticsearch/how-to-map-an-elasticsearch-index-using-the-python-client-266

I think we should start using a mapping for our index, this will give us much better search performance

Reply:
We already have a mapping for our index, as far as I can tell 

{
  "settings": {
    "index": {
      "number_of_shards": "1",
      "number_of_replicas": "1"
    }
  },
  "mappings": {
    "recipe": {
      "properties": {
        "image": {
          "type": "text",
          "fields": {
            "keyword": {
              "ignore_above": 256.0,
              "type": "keyword"
            }
          }
        },
        "link": {
          "type": "text",
          "fields": {
            "keyword": {
              "ignore_above": 256.0,
              "type": "keyword"
            }
          }
        },
        "rating": {
          "type": "float"
        },
        "ingredients": {
          "type": "text",
          "fields": {
            "keyword": {
              "ignore_above": 256.0,
              "type": "keyword"
            }
          }
        },
        "diet": {
          "type": "text",
          "fields": {
            "keyword": {
              "ignore_above": 256.0,
              "type": "keyword"
            }
          }
        },
        "id": {
          "type": "text",
          "fields": {
            "keyword": {
              "ignore_above": 256.0,
              "type": "keyword"
            }
          }
        },
        "title": {
          "type": "text",
          "fields": {
            "keyword": {
              "ignore_above": 256.0,
              "type": "keyword"
            }
          }
        },
        "total_time": {
          "type": "long"
        },
        "tags": {
          "type": "text",
          "fields": {
            "keyword": {
              "ignore_above": 256.0,
              "type": "keyword"
            }
          }
        }
      }
    }
  }
}